thunder/__init__.py

```diff
@@ -260,6 +273,7 @@ def jit(
     interpretation: None | INTERPRETATION_OPTIONS | str = None,
     cache: None | CACHE_OPTIONS | str = None,
     disable_torch_autograd: bool = False,  # TODO Revisit this UX for RC1
+    early_transforms: list | None = None,
```

Comment:
in what sense "early"?

Reply:
before the prologue is run. I'm starting to wonder whether we should do this to all transforms, but this is a second step.